api/register.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type RegUser struct {
```

Comment:
Using clear names doesn't hurt anyone!
use something like `RegisterForm` or `RegisterBody` or something like that